tests/functional/processes/file/test_fulfillment_process.py

```diff
@@ -0,0 +1,73 @@
+from processes import ClusterProcess, FulfillURLManifestProcess
+from model import Record, Item
+import json
+import os
+from sqlalchemy.orm import joinedload
+import pytest
+
+def test_fulfillment_process(db_manager, s3_manager, seed_test_data):
+
+    limited_access_record_uuid = seed_test_data['manifest_record_uuid']
+
+    cluster_process = ClusterProcess('complete', None, None, limited_access_record_uuid, None)
+    cluster_process.runProcess()
+    db_manager.session.commit()
+
+    record = db_manager.session.query(Record).filter_by(uuid=limited_access_record_uuid).first()
+    item = db_manager.session.query(Item)\
+        .options(joinedload(Item.links))\
+        .filter_by(record_id=record.id)\
+        .first()
+
+    assert item is not None, "Item not created by cluster process"
+    
+    epub_link = next((link for link in item.links if link.media_type == 'application/epub+zip'), None)
+    assert epub_link is not None, "EPUB link not found in item links"
+
+    test_manifest = {
+        "links": [{"type": epub_link.media_type, "href": epub_link.url}],
+        "readingOrder": [{"type": epub_link.media_type, "href": epub_link.url}],
+        "resources": [{"type": epub_link.media_type, "href": epub_link.url}],
+        "toc": [{"href": epub_link.url}]
+    }
```

Comment:
Is the test picking this manifest up? I think we may need to add some flags to these links.